subxt/src/utils/unchecked_extrinsic.rs

```diff
@@ -38,7 +68,7 @@ impl<Address, Call, Signature, Extra> scale_encode::EncodeAsType
         _types: &scale_info::PortableRegistry,
         out: &mut Vec<u8>,
     ) -> Result<(), scale_encode::Error> {
-        out.extend_from_slice(&self.0);
+        out.extend_from_slice(self.bytes());
```

Comment:
Or, `self.0.encode_to(type_id, types, out)` to fully defer to the internal impl :)